eclipse/ide-target-platform/category.xml

```diff
@@ -24,8 +24,8 @@
    <bundle id="org.hamcrest" version="1.1.0.v20090501071000"/>
    <bundle id="org.mortbay.jetty.server" version="6.1.26"/>
    <bundle id="org.mortbay.jetty.util" version="6.1.26"/>
-   <bundle id="com.google.guava" version="20.0"/>
-   <bundle id="com.google.gson" version="2.7"/>
+   <bundle id="com.google.guava" version="20.0.0"/>
```

Comment:
Why do we say 20.0 in pom.xml (and apparently must use 20.0 there, 20.0.0 doesn't work) and 20.0.0 here? Is the version in the OSGI bundle different than the maven version?